src/waveform/renderers/allshader/waveformrendermark.cpp

```diff
@@ -383,20 +388,22 @@ void allshader::WaveformRenderMark::drawUntilMark(const QMatrix4x4& matrix, floa
 // Note that in the legacy waveform widgets this is drawn directly
 // in the WaveformWidgetRenderer itself. Doing it here is cleaner.
 void allshader::WaveformRenderMark::updatePlayPosMarkTexture() {
-    float imgwidth;
-    float imgheight;
+    const float imgheight = m_waveformRenderer->getBreadth();
+    const float imgwidth = 11.f;
```

Comment:
```suggestion
    const float imgHeight = m_waveformRenderer->getBreadth();
    const float imgWidth = 11.f;
```
and where used, of course.